src/electionguard/serializable.py

```diff
@@ -86,14 +87,54 @@ def write_json_file(json_data: str, file_name: str, file_path: str = "") -> None
         json_file.write(json_data)
 
 
+ENCODE_THRESHOLD: Final[int] = 100_000_000
+
+
+def int_to_maybe_base64(i: int) -> Union[str, int]:
```

Comment:
This is an integer tool so this should be in the `.utils`

The name is also quite questionable. The word `maybe` should not be used in a method name.

Reply:
The use of `maybe` was to convey that small numbers go through while large numbers get encoded. I'm open to other names that convey this sentiment.

Reply:
Also, these functions are specifically for use in serialization, and not anywhere else. I don't buy the argument to move them.